test/aircraft/FlightManagementSystem/Fms.spec.js

```diff
@@ -654,14 +654,16 @@ ava('.replaceDepartureProcedure() does not call ._updateDepartureRunwayFromRoute
     t.deepEqual(responseForInvalidProcedure, expectedResponse);
 });
 
-ava('.replaceDepartureProcedure() calls ._updateDepartureRunwayFromRoute() when the departure procedure is applied successfully', (t) => {
+ava('.replaceDepartureProcedure() calls ._updateDepartureRunwayFromRoute() and updates route when the departure procedure is applied successfully', (t) => {
```

Comment:
With the changes to `. replaceDepartureProcedure()`, I would expect to see, at minimum, 8 new tests to cover each of the new logical branches.  This is one of the main reasons why we try to avoid `else` and `else if`, it adds unnecessary logical branches and makes code difficult to read.

Reply:
some tests have been added